src/components/drawer/Swipeable.tsx

```diff
@@ -303,8 +303,8 @@ export default class Swipeable extends Component<PropType, StateType> {
       }
     });
 
-    if (toValue === rowWidth * LEFT_TOGGLE_THRESHOLD && onToggleSwipeLeft) {
-      onToggleSwipeLeft({rowWidth, leftWidth});
+    if ((toValue === rowWidth * LEFT_TOGGLE_THRESHOLD || this.dragFired) && onToggleSwipeLeft) {
```

Comment:
At first I wasn't sure where this comes from (`this.dragFired`) I think the PR is missing some of the code changes. 
Anyway, the name of the variable was a little confusing to me. 
Is the flag purpose is to mark when the drag exceed threshold? if so, consider renaming it..

Reply:
Done.